src/test/java/uk/gov/hmcts/reform/bulkscanning/functionaltest/PaymentControllerFnTest.java

```diff
@@ -262,9 +263,9 @@ public void testNonMatchingPaymentsFromExelaThenBulkScan() throws Exception {
 
     @Test
     @WithMockUser(authorities = "payments")
-    public void testMatchingBulkScanFirstThenExela() throws Exception {
+    public void testMatchingBulkScanFirstThenExela() {
         //Request from Bulk Scan with one DCN
-        String dcn[] = {"111122223333888811111", "111122223333999911111"};
+        String[] dcn = {"111122223333888811111", "111122223333999911111"};
```

Comment:
This is definately more readable, thank you. Syntatically it doesn't really matter though, but a good style change.